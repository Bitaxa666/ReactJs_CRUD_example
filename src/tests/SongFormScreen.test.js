/**
 * Created by user on 4/4/18.
 */
import React from 'react';
import { configure, shallow, mount, render} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import SongForm from '../_components/SongForm';
import toJson from 'enzyme-to-json';

configure({ adapter: new Adapter() });


describe('SongForm', function() {

    it('render main layout ofr SongForm', () => {
        const wrapper = shallow(<SongForm />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });

});