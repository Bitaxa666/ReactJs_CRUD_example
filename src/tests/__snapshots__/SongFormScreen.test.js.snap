// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SongForm render main layout ofr SongForm 1`] = `
<div>
  <div
    className="ui container"
  >
    <form
      className="ui form"
      onSubmit={[Function]}
    >
      <h1>
        Add New Song
      </h1>
      <div
        className="field"
      >
        <label
          htmlFor="title"
        >
          Title
        </label>
        <input
          id="title"
          name="title"
          onChange={[Function]}
          value=""
        />
        <span />
      </div>
      <div
        className="field"
      >
        <label
          htmlFor="cover"
        >
          Cover URL
        </label>
        <input
          id="cover"
          name="cover"
          onChange={[Function]}
          value=""
        />
        <span />
      </div>
      <div
        className="field"
      />
      <div
        className="field"
      >
        <button
          className="ui primary button"
          disabled={true}
        >
          Save
        </button>
      </div>
    </form>
  </div>
</div>
`;
